Octokit.Tests/Clients/StarredClientTests.cs

```diff
@@ -346,19 +345,31 @@ public class TheGetAllForUserMethod
         public class TheGetAllStargazersForRepoMethod
         {
             [Fact]
-            public void RequestsCorrectUrl()
+            public async Task RequestsCorrectUrl()
```

Comment:
@ryangribble Yes, I tried to standardise all tests to use `async` and there is no failed tests without `async Task` return type. I've just tried be consiistent in sense that awaitable methods have to be called with usage of `await` keyword.

@shiftkey I don't understood your last sentence, do you mean that this change from `void` to `async Task` is going to produce some troubles with multithreading?